src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/xlf/DebugPropertyPage.xaml.ru.xlf

```diff
@@ -34,7 +34,7 @@
       </trans-unit>
       <trans-unit id="StringProperty|OpenLaunchProfilesEditor|Metadata|SearchTerms">
         <source>arguments;command line;working directory;environment variables;IIS;browser;URL;authentication;remote</source>
-        <target state="needs-review-translation">аргументы;командная строка;рабочая папка;переменные среды</target>
+        <target state="new">arguments;command line;working directory;environment variables;IIS;browser;URL;authentication;remote</target>

```

Comment:
These also appear to have been reverted. The source string did change, but this throws away the previous translation which may make a new translation less consistent with what was there before, not to mention making more work for the translators.

Reply:
It doesn't throw away the translations for the translators. Always keep in mind that the translated XLF files are the *result* of the pipeline, not the *source*. From the latest LEGO PR, you can see they still have the source information they had before. Once this LEGO PR is merged, it'll run again and spit out new one of these PRs with the translations. I think it was just a timing thing that these PRs happened just before the LEGO PRs.
![image](https://user-images.githubusercontent.com/17788297/122996699-03dbf500-d360-11eb-9484-a2d8dc8d708d.png)

https://github.com/dotnet/project-system/pull/7347

Reply:
Is there a scenario where we need to review these PRs then, beyond checking for system failure?